edx_proctoring/static/proctoring/js/views/proctored_exam_add_allowance_view.js

```diff
@@ -11,7 +11,9 @@ edx = edx || {};
         template: null,
         template_url: '/static/proctoring/templates/add-new-allowance.underscore',
         initialize: function(options) {
-            this.proctored_exams = options.proctored_exams;
+            this.all_exams = options.proctored_exams;
```

Comment:
Instead of changing on the existing view, you would need to copy and past the code here into a new view file.
Keep the old view file unchanged.
At the place where the view is instantiated, check the waffle flag passed from the server, and render the different view based on waffle's flag's value